src/transactions/transactions.model.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  Model,
+  Column,
+  ForeignKey,
+  Table,
+} from 'sequelize-typescript';
+import { Card } from '../cards/card.model';
+import { DataTypes } from 'sequelize';
+import { ApiProperty } from '@nestjs/swagger';
+
+
+
+interface TransactionCreateAttrs {
+  sender_card_id: number;
+  sender_full_name: string;
+  receiver_card_id: number;
+  receiver_card_number: string;
+  receiver_full_name: string;
+  transaction_amount: number;
+  transaction_description: string;
+  transaction_type: string;
+}
+
+@Table({ tableName: 'Transaction' })
+export class Transaction extends Model<Transaction, TransactionCreateAttrs> {
```

Comment:
Not sure what '@ApiProperty' does , but unless its adding property to the very ,model , this extension violate Open/Closed principle. You use it often through your code , so if its actually wrong - change it.
```suggestion
export class Transaction {
//your code here
}
```